CardsCashCasino/Data/Constants.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace CardsCashCasino.Data;
+
+public class Constants
+{
+    public const int AI_PLAYER_COUNT = 4;
```

Comment:
Add a summary comment. Since the AI player count is just used for poker, we can use the comment to identify that.